@vates/disposable/debounceResource.test.js

```diff
@@ -1,16 +1,19 @@
 'use strict'
 
-/* eslint-env jest */
+// eslint-disable-next-line n/no-unpublished-require
+const { describe, it } = require('test')
+// eslint-disable-next-line n/no-unpublished-require
```

Comment:
Do you have the same issue with `sinon`? :thinking:

Reply:
Yes I do. I don't know why :/

Reply:
I'm investigating.

Reply:
Should be fixed: https://github.com/vatesfr/xen-orchestra/commit/551670a8b92cb609dcd887d0b39213972caa5449

Reply:
Great!

Do you want me to remove comments then?

Reply:
Yes please, if too hard to do for all your PRs, we'll do it in a single commit after merging them.

Reply:
Yes, I'll do it after the merges. It will be more convenient